.github/workflows/publish.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish
+
+on:
+  release:
+    type: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: jerray/publish-docker-action@master
```

Comment:
You think you could give a try and see if you can write this in our action instead of using another one? It seems a bit overkill to use that when what we really need is just build the docker image and push it.

Great opportunity to learn more about Docker in case you don't already ofc :)

Reply:
Sure, I choose to give `jerray/publish-docker-action` workflow a try because it handle tags so smoothly.

But it does not matter now :)